content/en/platform/corda/5.2/developing-applications/cordapp-template/overview/_index.md

```diff
@@ -23,16 +24,18 @@ On the left, you can see the folder structure created, ready for CorDapps develo
 
 For Kotlin, write your {{< tooltip >}}flow{{< /tooltip >}} code in `workflows/src/main/kotlin/<your package path>` and your contract and {{< tooltip >}}states{{< /tooltip >}} code in `/contracts/src/main/kotlin/<your package path>`.
 
-For Java, use `workflows/src/main/java/<your package path>` and your contract and states code in `/contracts/src/main/java/<your package path>`.
+For Java, use `workflows/src/main/java/<your package path>` for your flows and `/contracts/src/main/java/<your package path>` for your contract and states code.
 
 For test code, use the corresponding test folder.
+
 ## Gradle Helpers for the Combined Worker
 
-On the right, you can see the Gradle tasks that we have included to help you work with a local deployment of Corda using the combined worker:
-{{< figure src="gradle-helpers.png" figcaption="CSDE gradle helpers" alt="CSDE gradle tasks in IntelliJ" width=100% length=100% >}}
+On the right, you can see the Gradle tasks that help you work with a local deployment of Corda using the combined worker:
+{{< figure src="gradle-helpers.png" figcaption="Corda Runtime gradle helpers" alt="Corda Runtime gradle tasks in IntelliJ" width=100% length=100% >}}
 The **combined worker** is a Corda cluster that runs all of the workers in one JVM process.
 
 The helpers are split into three folders:
+
 * [csde-corda](#csde-corda)
```

Comment:
From here down, we're going to need some new text.

- We're no longer using the JAR directly, but a docker container for each component.
- task groups have been renamed
- individual tasks have been renamed
<img width="346" alt="Screenshot 2024-02-28 at 10 24 46" src="https://github.com/corda/corda-docs-portal/assets/92731849/08ecec3c-cfb5-416f-bd7a-cfb1d81cd332">

Reply:
https://github.com/corda/corda-docs-portal/pull/2060/commits/38e04e083215bcac337e9a40b018facee7255a3b